app/helpers/alchemy/elements_helper.rb

```diff
@@ -224,14 +229,17 @@ def element_tags_attributes(element, options = {})
     #
     # @param [Array] elements - The elements you want to sort
     # @param [String] content_name - The name of the content you want to sort by
+    # @param [Boolean] reverse - Reverse the sorted elements order
     #
     # @return [Array]
     #
-    def sort_elements_by_content(elements, content_name)
-      elements.sort_by do |element|
+    def sort_elements_by_content(elements, content_name, reverse = false)
+      sorted_elements = elements.sort_by do |element|
         content = element.content_by_name(content_name)
         content ? content.ingredient.to_s : ''
       end
+
+      reverse ? sorted_elements.reverse : sorted_elements
```

Comment:
Ok, we can leave in this place. But, we should rename the method (with an alias to the former name), just because the name doesn't reflect this method any more. Is `sorted_elements` a good name?